src/data/equiv/mul_add.lean

```diff
@@ -584,10 +584,17 @@ end equiv
 `mul_equiv.inv' G : G ≃* Gᵐᵒᵖ` for the non-commutative case. -/
 @[to_additive "When the `add_group` is commutative, `equiv.neg` is an `add_equiv`."]
 def mul_equiv.inv (G : Type*) [comm_group G] : G ≃* G :=
-{ to_fun := has_inv.inv,
-  inv_fun := has_inv.inv,
+{ to_fun   := has_inv.inv,
+  inv_fun  := has_inv.inv,
   map_mul' := mul_inv,
-  ..equiv.inv G}
+  ..equiv.inv G }
+
+/-- When the group with zero is commutative, `equiv.inv₀` is a `mul_equiv`. -/
+def mul_equiv.inv₀ (G : Type*) [comm_group_with_zero G] : G ≃* G :=
```

Comment:
Could you please provide simp lemmas for this equiv? So that `mul_equiv.inv₀ x` get's simped to `x⁻¹`.
Also `mul_equiv.inv₀.symm = mul_equiv.inv₀`.

(Arguably, this should also be done for the equiv above this one, but maybe that's not the job of this PR.)

Reply:
Do you mean these two lemmas?
```
@[simp] lemma mul_equiv.inv₀.def (G : Type*) [comm_group_with_zero G] {x : G} :
  mul_equiv.inv₀ G x = x⁻¹ := rfl

@[simp] lemma mul_equiv.inv₀.symm (G : Type*) [comm_group_with_zero G] {x : G} :
  (mul_equiv.inv₀ G).symm = mul_equiv.inv₀ G := rfl
```
What would be a good name for the second one?

Reply:
Some tweaks:

* underscores not dots
* x isn't used in the second lemma

```lean
@[simp] lemma mul_equiv.inv₀_apply {G : Type*} [comm_group_with_zero G] {x : G} :
  mul_equiv.inv₀ G x = x⁻¹ := rfl

@[simp] lemma mul_equiv.inv₀_symm (G : Type*) [comm_group_with_zero G] :
  (mul_equiv.inv₀ G).symm = mul_equiv.inv₀ G := rfl
```

Does `@[simps apply]` generate the first lemma for you?

Reply:
Thanks for the feedback; I've just committed these changes.